pkgs/by-name/ya/yandex-ffmpeg-codecs/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  unzip,
+  fetchurl,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yandex-ffmpeg-codecs";
+  version = "0.89.0";
+
+  src = fetchurl {
+    url = "https://github.com/nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt/releases/download/${version}/${version}-linux-x64.zip";
+    hash = "sha256-uaCGZsPQrA1dl90IqRg7TgWMJyQAGBBF/qNBTShVHcY=";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  dontStrip = true;
+  stripDebugList = [ "." ];
+
+  unpackPhase = ''
+    mkdir $TMP/ffmpeg-codecs/ $out/lib/ -p
+    unzip -X -d $TMP/ffmpeg-codecs/ $src
+  '';
```

Comment:
When you add unzip to `nativeBuildInputs`, it should unpack this automatically.

Reply:
unzip is added in nativeBuildInputs list.
but will try with new change to reduce this multiple folders.